umbraco/Umbraco-CMS
# v9 - Fix unable to authorize upgrade from versions < 9.3.0

Should close https://github.com/umbraco/Umbraco-CMS/issues/12389

Steps to test

1. Create an Umbraco 9.2.0 site and install to create db
2. Confirm can authorize upgrade using web.ui from this branch pointed at db from step 1.
